testsuite/libra-fuzzer/README.md

```diff
@@ -22,7 +22,7 @@ a corpus with `cargo run --bin libra-fuzzer generate <target>`.
 Once a corpus has been generated, the fuzzer is ready to use, simply run:
 
 ```
-RUSTC_BOOTSTRAP=1 cargo run --bin libra-fuzzer fuzz <target> -- --release --debug-assertions
+RUSTC_BOOTSTRAP=1 cargo run --bin libra-fuzzer --release fuzz <target>
```

Comment:
can you actually use `--debug` instead :D? The rationale could be included in the README as well: we want a number of checks to be enabled when we fuzz. For example, [overflow-checks](https://rust-lang.github.io/rfcs/1535-stable-overflow-checks.html).

Reply:
definitely a trade-off here, in `--debug` mode we'll have much lower iters/sec, cover less of the search space, and probably miss more hard-to-find bugs. the binary will also look more different than prod, which runs in release mode

Reply:
yeah, we've talked about that before and we chose to fuzz things in debug mode instead. We are definitely missing on a bunch of bugs if we don't do this :/ we could run fuzzers in both mode as well...